### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#425] [Esami & Esoneri M-Z]
## Traccia

<img width="929" alt="image" src="https://github.com/user-attachments/assets/db6c6ba0-0c4e-4564-b9bd-c87bcc543b01">

> Questo esercizio è stato proposto **anche** come **Esercizio 3** della traccia di esame **del 22 gennaio 2019**

**Comment by rimaout** [comment]
<img width="700" alt="image" src="https://github.com/user-attachments/assets/073de08e-3afa-48a9-b3f1-e528243af207">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [*20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/e7be5139-661a-49a6-877e-a190dcf66310)

\* _Questo esercizio era in comune rispetto alla traccia del 22 gennaio 2019, gli esercizi 7, 8 e 9 sono specifici delle due rispettive tracce, per quelli vedi [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf)_
